test/util/common.js

```diff
@@ -113,6 +117,20 @@ common.forValue = async function(obj, key, val, timeout = 30000) {
   });
 };
 
+common.constructBlockMiner = function (node, nclient) {
```

Comment:
I can _feel_ others thinking this is less than optimal - the whole "pass me a full node and api client" signature is wonky 🤔

Reply:
I can't think of a better way besides using a closure like this, can you?

Reply:
Nope!